cross/bison/Makefile

```diff
@@ -12,7 +12,10 @@ COMMENT  = General-purpose parser generator
 LICENSE  = GPLv3
 
 GNU_CONFIGURE = 1
-ifeq ($(findstring $(ARCH), powerpc ppc824x ppc853x ppc854x),$(ARCH))
+
+include ../../mk/spksrc.archs.mk
```

Comment:
`spksrc.archs.mk` *is included* in `spksrc.cross-cc.mk` and other mk files (all that include `spksrc.common.mk`).
But sometimes you need to have the ARCHS definitions available before the include of `spksrc.cross-cc.mk` and for such situations I prefer to include the new `spksrc.archs.mk` instead of `spksrc.common.mk`. And from the filename you can see that such includes are for arch definitions only.